shared-config/iptables.conf

```diff
@@ -0,0 +1,26 @@
+# Generated by iptables-save v1.8.9 (nf_tables) on Tue Jul 30 11:01:37 2024
+*raw
+:PREROUTING ACCEPT [116987909:15570482967]
+:OUTPUT ACCEPT [100653555:117594101415]
+:myrain - [0:0]
+-A PREROUTING -i eth0 -j myrain
+# unknown? (Hetzner IP)
+-A myrain -s 88.198.148.245/32 -p tcp -m tcp --dport 22 -j ACCEPT
```

Comment:
This is the old IP address of `php-jump2.php.net` (https://wiki.php.net/systems/php-jump2). That rule can most likely be removed.